packages/backend/src/apps/discord/actions/create-new-event/index.ts

```diff
@@ -0,0 +1,152 @@
+import defineAction from '../../../../helpers/define-action';
+
+export default defineAction({
+  name: 'Create a new event',
+  key: 'createNewEvent',
+  description: 'Creates a new event',
+  arguments: [
+    {
+      label: 'Type',
+      key: 'entity_type',
+      type: 'dropdown' as const,
+      required: true,
+      description: 'The name of the event.',
+      variables: true,
+      options: [
+        {label: 'Stage-Channel', value: 1},
+        {label: 'Voicechannel', value: 2},
+        {label: 'External', value: 3}
+      ]
+    },
+    {
+      label: 'Privacy level',
+      key: 'privacy_level',
+      type: 'dropdown' as const,
+      required: true,
+      description: 'The privacy level of the event',
+      variables: true,
+      options: [
+        {label: 'Public', value: 1},
```

Comment:
I couldn't find any documentation reference for the public privacy level. When I try to create a public scheduled event, it responds with the error below.

```
{
  "privacy_level": [
    "Invalid privacy level for event."
  ]
}
```

Could you share some resources with me? @felifluid 

Asking in case this is not feasible, we may well remove the argument and always have `2` for `privacy_level` in the payload.

Reply:
Oh nice spot. Looking at the [docs of the privacy level](https://discord.com/developers/docs/resources/guild-scheduled-event#guild-scheduled-event-object-guild-scheduled-event-privacy-level) the only accepted value at this moment is `2`. So yes you might as well remove the argument and always set this to `2` in the payload.